.github/ISSUE_TEMPLATE/refactor.yml

```
@@ -1,7 +1,23 @@
-name: Code Refactor
-about: Template for creating issues related to code refactoring
+#
```

yamllint (1.35.1)

[error] wrong new line character: expected \n

(new-lines)